src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/RunConcurrentRepresentationTests.kt

```diff
@@ -446,7 +445,7 @@ class ThreadPoolRunConcurrentRepresentationTest : BaseRunConcurrentRepresentatio
 class CoroutinesRunConcurrentRepresentationTest : BaseRunConcurrentRepresentationTest<Unit>(
     "run_concurrent_test/coroutines/coroutines"
 ) {
-    // TODO: check why flakiness appears in this test
+    // TODO: coroutine names are non-deterministic
```

Comment:
But you specify deterministic thread names and not coroutine names here. Could you please elaborate on the `todo` so that the problem is clearer? Taking a look at the issue does not help(

Reply:
I extended the TODO comment:

```
// TODO: coroutine names are still non-deterministic.
//   Apparently, for some reason the `kotlinx.coroutines` is run in DEBUG mode,
//   and in this mode `COROUTINE_ID` is appended to each coroutine's name.
//   The `COROUTINE_ID` is global `AtomicLong` which is incremented on each new coroutine creation,
//   and thus is non-deterministic.
```